patches/pyodide@0.23.4.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/pyodide.d.ts b/pyodide.d.ts
+index d5ed46f6345855a75ec6f2b7ef73237a0af64a7e..e0087c83792558ca30713e9d07a9a37625f68d8d 100644
+--- a/pyodide.d.ts
++++ b/pyodide.d.ts
+@@ -1118,6 +1118,15 @@ export declare function loadPyodide(options?: {
+ 	 * (``pyodide.js`` or ``pyodide.mjs``) removed.
+ 	 */
+ 	indexURL?: string;
++	/**
```

Comment:
[This code did not get (accidentally) picked for the 0.23.4 release](https://github.com/pyodide/pyodide/pull/3972/files#r1255398268). but since this is only needed for development and building, keeping this change in a patch should be okay for now, until pyodide releases the next patch.